src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1153,20 +1153,30 @@ internal static TResult ExecuteScriptAndGetItem<TResult>(string scriptToExecute,
         /// <returns>A Task that can be awaited for completion.</returns>
         public async Task LoadHostProfilesAsync()
         {
-            if (this.profilePaths != null)
+            if (this.profilePaths == null)
             {
-                // Load any of the profile paths that exist
-                var command = new PSCommand();
-                foreach (var profilePath in GetLoadableProfilePaths(this.profilePaths))
-                {
-                    command.AddCommand(profilePath, false).AddStatement();
-                }
-                await ExecuteCommandAsync<object>(command, sendOutputToHost: true).ConfigureAwait(false);
+                return;
+            }
 
-                // Gather the session details (particularly the prompt) after
-                // loading the user's profiles.
-                await this.GetSessionDetailsInRunspaceAsync().ConfigureAwait(false);
+            // Load any of the profile paths that exist
+            var command = new PSCommand();
+            bool hasLoadablePath = false;
+            foreach (var profilePath in GetLoadableProfilePaths(this.profilePaths))
```

Comment:
Oh, I forgot. I was seeing an exception here in the debugger because none of the profile files existed. And it was my change that caused it in another PR. So I fixed it.